SpatialGDK/Documentation/content/game-client-interest-management.md

```diff
@@ -0,0 +1,115 @@
+<%(TOC)%>
+# Game client interest management
+
+The GDK uses [interest]({{urlRoot}}/content/glossary#interest) to control the information that [game clients]({{urlRoot}}/content/glossary#client-workers) receive about the world around them.
+
+Game clients need information about Actors that they don’t own (see the Unreal documentation on [owning](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/OwningConnections) to help them to correctly manipulate the Actors that they _do_ own, and to render the SpatialOS world. Interest enables game clients to receive the relevant information.
+
+You can define your game client’s interest in three ways, which you can use alongside each other:
+
+* **`NetCullDistanceSquared`**<br>
+A property that you define on an Actor’s class. For any Actor of that class or a derived class, game clients whose players are within the distance you specify receive updates about the Actor.<br>
+* **`ActorInterestComponent`**<br>
+An Unreal Actor component that you add to an Actor that a game client might own. This includes Actors that are possessed by Actors that the game client owns. For example, if a game client owns a PlayerController, and the PlayerController possesses a Pawn, then the game client owns the Pawn.
+
+    Within this component, you provide a list of queries that express the Actors that you want the game client to receive updates about. This is a more granular way of setting up interest than `NetCullDistanceSquared`.<br>
+* **`AlwaysInterested`**<br>
+A `UPROPERTY` that you add to an Actor that a game client might own. It refers to a specific Actor that you want the game client to always receive updates about.
+
+> **Tip**: All three ways of defining interest can coexist. However, if you've specified all of the interest in your game using `ActorInterestComponent` and `AlwaysInterested`, you can turn off `NetCullDistanceSquared` using the toggle in `ActorInterestComponent`.
```

Comment:
I think maybe this tip should be moved into `## ActorInterestComponent` and should only talk about how there's a toggle rather than referring to "AlwaysInterested" but we should talk about this.